src/Microsoft.AspNetCore.Http.Extensions/UriHelper.cs

```diff
@@ -101,7 +101,7 @@ public static string Encode(Uri uri)
         /// <returns></returns>
         public static string GetEncodedUrl(this HttpRequest request)
```

Comment:
Encoded in what way and for what purpose? This method does a particular type of encoding for use in Http headers that is not applicable elsewhere.